app/routes/validation_helper.py

```diff
@@ -0,0 +1,15 @@
+from flask import abort, make_response
+
+
+def get_valid_item_by_id(model, id):
```

Comment:
I love that you broke out a separate file for this and for your different routes. 😄